frameioclient/client.py

```diff
@@ -243,6 +243,16 @@ def get_comments(self, asset_id, **kwargs):
     endpoint = '/assets/{}/comments'.format(asset_id)
     return self._api_call('get', endpoint, **kwargs)
 
+  def get_replies(self, comment_id, **kwargs):
+    """
+    Get replies for the comment.
+
+    :Args:
+      comment_id (string): The comment id.
+    """
+    endpoint = '/comments/{comment_id}/replies'.format(comment_id=comment_id)
```

Comment:
This is a great addition, however I'd like to stick to our current codestyle and have it look  more like:

`endpoint = '/comments/{}/replies'.format(comment_id)`

Let me know what you think, and assuming you agree - can you make that change and then we'll get this PR merged in!

Reply:
Check it now!

Reply:
Looks good to me @makseq! 

I believe I just took care of this in another PR as I was cleaning things up so this might cause merge conflicts now, but I'm going to see if I can find a way to dovetail this in there so you get some contributor credit!

Thanks again for the contribution :)

Reply:
Credit is cool! Frame.io is expensive for my cinema hobby :(